pkgs/development/tools/database/mermerd/default.nix

```
@@ -0,0 +1,29 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "mermerd";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "KarnerTh";
+    repo = "mermerd";
+    rev = "v${version}";
+    sha256 = "sha256-mQIHv3KjUCtXhzbqBNuooCH7b/LXVUHTZO5n0CJym/A=";
+  };
+
+  vendorHash = "sha256-h1jSPJYFJKkP8kNfD8bn8cmaqVZ20K+S6czau8DnauQ=";
+
+  # the tests expect a database to be running
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Create Mermaid-Js ERD diagrams from existing tables";
+    homepage = "https://github.com/KarnerTh/mermerd";
```

Tools (e.g., `nix-update`) could add a link to the changelog in the commit messages and/or the PR description if `changelog = ....` is part of meta.

```suggestion
    homepage = "https://github.com/KarnerTh/mermerd";
    changelog = "https://github.com/KarnerTh/mermerd/releases/tag/v${version}";
```